async-profiler.graphqls

```diff
@@ -0,0 +1,176 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# use to create profiler task
+input AsyncProfilerTaskCreationRequest {
+    # need to monitor service id
+    serviceId: ID!
+    # serviceInstanceIds to monitored under the special service.
+    serviceInstanceIds: [String!]!
+    # duration of this task (second)
+    duration: Int!
+    # when the segment starts to execute, how long must it take before the monitor can be enable
+    events: [AsyncProfilerEventType!]!
+    # other async-profiler args example alloc=2k,lock=2s
+    execArgs: String
+}
+
+# profiler task create result
+type AsyncProfilerTaskCreationResult {
+    # if null or empty means the task create success, otherwise get create error reason
+    errorReason: String
```

Comment:
How many kinds of errors we will mean? If there are several, we should consider the error codes.

Reply:
There are about four types. Here, errorReason is mainly used to return the parameter that is not standardized, or there is a task being executed at the same time.

Reply:
Let's provide an enum to list the error code(extend later). Then, message could be about details.